build-logic/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

@alexvanyo I was able to get it to work by just having the two lines below in the same `gradle-wrapper.properties` file without downloading another `gradle-wrapper.jar`

```suggestion
jarFile = "${project.projectDir}/gradle/wrapper/gradle-wrapper.jar" // archive
scriptFile = "${project.projectDir}/gradlew" // wrapper script
```

---

When I had those two lines (after cleaning the project, and reopening `build-logic` in AS), I got an error of `Could not load wrapper properties`